### ISSUE TITLE
word-level formatting with valign not respected

### ISSUE THREAD

**Issue by SunilAZ** [gitbrent/PptxGenJS#1051]
### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [3.9.0]

Please specify what version(s) of PowerPoint you are targeting: [not_related]

### Expected / Desired Behavior / Question

```
        slide.addText(
         [ 
          {text: "1", options: {align: "center", valign: "center"}},
          {text: "2", options: {align: "right", valign: "bottom"}}
        ],
        {shape: pptx.ShapeType.rect, valign: "center"}
       )
```

For the above code, I'd expect a single textbox to be created with:
- "1" centred horizontally and vertically
- "2" placed in the bottom right corner

### Observed Behavior

Instead, "1" is centred correctly but "2" is placed to the right but is vertically centered.

It may be likely that the defaut value of `valign` of the root textbox is favoured over the text sub-elements `valign` value here, but I believe this is a bug.

**Comment by gitbrent** [comment]
I do not believe a TextBox in PowerPoint can have valign settings per line/text run, only a single vertical setting exists.

If you can create such a shape in PowerPoint, please provide details.
